src/main/java/com/tomtom/services/configuration/implementation/Configuration.java

```diff
@@ -85,14 +85,25 @@ public Configuration(@Nonnull final ConfigurationServiceProperties configuration
      * @param configurationServiceProperties Configuration.
      * @param overrideStartupConfiguration   String configuration which overrides the configuration URL.
      *                                       Note that the regular constructor uses this as well.
+     @Nullable
```

Comment:
Hmm... I have no idea why this is here. Let me remove this before you integrate.